gui/index.html

```diff
@@ -846,6 +853,25 @@ <h4 translate class="modal-title">Settings</h4>
     </div>
   </div>
 
+  <!-- Error Logs Sending modal -->
+
+  <div id="el" class="modal fade" data-backdrop="static" data-keyboard="false" tabindex="-1">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="modal-header alert alert-success">
+          <h4 translate class="modal-title">Allow Error Log Sending To Development Team?</h4>
+        </div>
+        <div class="modal-body">
+          <p translate>The anonymous error log will be send to development team everytime error occurs. It will be used to repair bugs and monitor problematic issues.</p>
```

Comment:
send -> sent
everytime -> every time

Potential addition: "Syncthing is a community project and this is a good first step to get involved and help the project."